lib/docusign_rest/client.rb

```diff
@@ -67,6 +70,10 @@ def headers(user_defined_headers={})
         'Accept' => 'json' #this seems to get added automatically, so I can probably remove this
       }
 
+      if access_token.is_a?(Proc)
```

Comment:
What handles the non-Proc case here?

Reply:
It's usually just a string set in a config